src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -143,17 +144,27 @@ def validate_request_create_args(request_args, config, reqmgr_db_service, *args,
     permission = getWritePermission(request_args)
     authz_match(permission['role'], permission['group'])
 
-    # set default values for teh request_args
+    # load the correct class to in order to validate the arguments
     specClass = loadSpecClassByType(request_args["RequestType"])
-    setArgumentsWithDefault(request_args, specClass.getWorkloadArguments())
 
-    # get the spec type and validate arguments
-    spec = loadSpecByType(request_args["RequestType"])
     if request_args["RequestType"] == "Resubmission":
-        initialize_resubmission(request_args, config, reqmgr_db_service)
+        # do not set default values for Resubmission since it will be inherited from parent
+        # both create & assign args are accepted for Resubmission creation
+        acceptedArgs = specClass.getWorkloadCreateArgs()
```

Comment:
And this makes sense too. We can load the original spec definition and then update it with the resubmission one.
I'll make these changes today.